src/constants/concepts.js

```diff
@@ -3,3 +3,4 @@ export const moviesConstant = 'movies'
 export const movieConstant = 'movie'
 export const dashboardConstant = 'dashboard'
 export const watchlistConstant = 'watchlist'
+export const favoritesConstant = 'favorites'
```

Comment:
better not to use suffixes like `constant`, `object`, `function` for variable names